Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM golang:1.15.3-buster AS builder
-RUN apt-get update -y
-RUN apt-get install -y libgpgme-dev
-WORKDIR /go/src/tagger
+#
+# Builder
+#
+
+FROM registry.fedoraproject.org/fedora:latest AS builder
+
+WORKDIR /src
 COPY . .
-RUN go build -o /tagger ./cmd/tagger
 
+RUN yum install -y \
+    golang \
```

Comment:
What is the golang version that comes with this yum install? I have a few changes coming in that require go 1.16 (embedding some static text files). Another thing is use `dnf` instead of `yum`

Reply:
Here we have the latest golang version in Fedora, however, for a next interaction we can make sure the Golang is exactly what's expected. I wonder if we have a base image with Golang and Fedora.

Reply:
Well, it is not go 1.16 :-(

```
STEP 5: RUN make
go build -o _output/bin/tagger ./cmd/tagger
go build -o _output/bin/kubectl-tag ./cmd/kubectl-tag
cmd/kubectl-tag/static/static.go:6:2: cannot find package "." in:
        /src/vendor/embed
make: *** [Makefile:28: kubectl-tag] Error 1
STEP 6: FROM registry.fedoraproject.org/fedora:latest
Error: error building at STEP "RUN make": error while running runtime: exit status 2
make: *** [Makefile:57: image] Error 125
```

You may not be seeing this error because the `main` branch where you based this PR did not use the `embed` package.

Reply:
Indeed, we can have newest Golang by installing it from rawhide, we should see it landing on Fedora 33 in the next few days.
By the way, the Fedora version from registry.fedoraproject.org was 32, so moving it to docker.io, where we have 33 available.

Reply:
Excellent. All changes you proposed here are a must, amazing. We are good to go, just one last thing before the merge: by moving the COPY . . directive to after the dnf foo magic  won't give us a speed bump when building locally ? (cache and stuff).

Reply:
Sure, moving the `COPY` and `make` commands closer makes more sense for this situation, please consider latest changes pushed.

Reply:
Excellent!